connection/pulsar_http_conn.go

```diff
@@ -0,0 +1,53 @@
+package connection
+
+import (
+	"encoding/json"
+	"github.com/flipkart-incubator/go-dmux/core"
+	sink "github.com/flipkart-incubator/go-dmux/http"
+	source "github.com/flipkart-incubator/go-dmux/pulsar"
+	"log"
+)
+
+// PulsarConnConfig holds config to connect pulsar source to http sink
+type PulsarConnConfig struct {
+	Dmux        core.DmuxConf     `json:"dmux"`
+	Source      source.PulsarConf `json:"source"`
+	Sink        sink.HTTPSinkConf `json:"sink"`
+	PendingAcks int               `json:"pending_acks"`
+}
+
+// PulsarConn abstracts connection
+type PulsarConn struct {
+	EnableDebugLog bool
+	Conf           interface{}
+}
+
+// getConfiguration parses configs and returns connection config
+func (c *PulsarConn) getConfiguration() *PulsarConnConfig {
+	data, _ := json.Marshal(c.Conf)
+	var config *PulsarConnConfig
+	json.Unmarshal(data, &config)
```

Comment:
Why marshal and unmarshal again? Can please change the type of Conf to PulsarConnConfig to remove this